docs/src/docusaurus-tree-sitter-plugin/index.js

```diff
@@ -6,6 +6,7 @@
 
 module.exports = function () {
   return {
+    name: "tree-sitter",
```

Comment:
Did this sneak in from somewhere or is it related to other docs changes?

Reply:
QoL issue.... I hit this when I thought I needed to upgrade our Docusaurus version, and it errored out due to this missing `name` property. Since I was there, figured I'd fix so future Docusaurus upgraders don't need to.

Reply:
Thanks, good (intentional) change :)